spec/controllers/api/v1/rents_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe Api::V1::RentsController do
+  include_context 'Authenticated User'
+
+  describe 'GET #index' do
+    let!(:rents) { create_list(:rent, 3, user: user) }
+
+    before { get :index, params: { user_id: user.id } }
+
+    it { expect(response).to have_http_status(:ok) }
+
+    it 'responds with the user\'s rents in json format' do
+      expected = ActiveModel::Serializer::CollectionSerializer.new(
+        rents, each_serializer: RentSerializer, root: false
+      ).to_json
+
+      expect(response_body.to_json) =~ JSON.parse(expected)
+    end
+  end
+
+  describe 'POST #create' do
+    let!(:book) { create(:book) }
```

Comment:
Use lazy load (e.g., `let(:book) ...`).
You can read ["Use let and let!"](http://www.betterspecs.org).